app/controllers/sidebar_sections_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class SidebarSectionsController < ::ApplicationController
+  requires_login
+
+  def create
+    sidebar_section =
+      SidebarSection.create!(
+        section_params.merge(user: current_user, sidebar_urls_attributes: links_params),
+      )
+
+    render json: SidebarSectionSerializer.new(sidebar_section)
+  rescue ActiveRecord::RecordInvalid => e
+    render_json_error(e.record.errors.full_messages.first)
+  end
+
+  def update
+    sidebar_section = SidebarSection.find(section_params["id"])
```

Comment:
`#find` will raise an error if the record does not exist. I think we might want to use `find_by` here instead and return a `404`?

Reply:
What do you think would be a benefit of being graceful? I can return an error that "Section does not exists" but it is kind of edge case. I think that if might happen only when someone is tempering params or old tab is opened

Reply:
One big benefit of being graceful is that we don't flood our error logs with unnecessary error. Also, we get to return a proper response code instead of 500 which can be confusing for the end user.